examples/easy-usage.cpp

```diff
@@ -1,5 +1,6 @@
 
-#include "chickenHook/chickenhook.h"
+#include "/src/include/chickenHook/hooking.h"
```

Comment:
While using our artifacts, you'll need to use 

#include <chickenHook/hooking.h>

please change it to this